src/java.base/unix/classes/sun/nio/fs/UnixFileStore.java

```diff
@@ -172,6 +172,27 @@ public Object getAttribute(String attribute) throws IOException {
         throw new UnsupportedOperationException("'" + attribute + "' not recognized");
     }
 
+    // returns true if extended attributes enabled on file system where given
+    // file resides, returns false if disabled or unable to determine.
+    protected boolean isExtendedAttributesEnabled(UnixPath path) {
```

Comment:
This used to be private method in LinuxFileStore, now it's a protected method so might be cleaner to change the method description to use /** .. */ comment as it's used from sub-classes.